tests/utils.go

```diff
@@ -4870,6 +4870,12 @@ func SkipSELinuxTestIfRunnigOnKindInfra() {
 	}
 }
 
+func SkipMigrationKillertIfRunningOnKindInfraIPv6() {
+	if IsRunningOnKindInfraIPv6() {
+		Skip("Skip Migration killer tests till issue https://github.com/kubevirt/kubevirt/issues/4086 is fixed")
```

Comment:
killer tests? What is that?

Reply:
see please the test where its used, this is a new test that have a migration killer pod which kills the virt-handler
https://github.com/kubevirt/kubevirt/pull/4087/files#diff-d642ef9101fdb0671d818d73a1c3f977R1177

Reply:
The test has an It with text in it, that is the test name.
SkipFailMigrationTestOn... is more appropriate (or anything similar), and we are not killing anyone in the process. :)

Please adjust the commit message as well.

Reply:
done PTAL

Reply:
Still one killer left...

Reply:
done